deployment/integration-tcs/gateway-TEST-WITH-TCS.md

```diff
@@ -0,0 +1,48 @@
+```sh
+
+# Install TCS Signer
+export CA_SIGNER_NAME=sgx-signer
+cat << EOF | kubectl create -f -
+apiVersion: tcs.intel.com/v1alpha1
+kind: TCSClusterIssuer
+metadata:
+    name: $CA_SIGNER_NAME
+spec:
+    secretName: ${CA_SIGNER_NAME}-secret
+    selfSign: false
+EOF
+
+# Get the CA Cert, then export the CA Cert in CACERT and replace it in istio-hsm-config.yaml caCertificates field
+$(kubectl get secret -n tcs-issuer ${CA_SIGNER_NAME}-secret -o jsonpath='{.data.tls\.crt}' |base64 -d | sed -e 's;\(.*\);        \1;g')
+
+# Create the CA certificate file
+echo $CACERT > cacert.crt
+
```

Comment:
Is the `CACERT `here the same as the TCS signer cert?

Reply:
Yeah, that's my question for @irisdingbj and you, I think there are 2 places to obtain the cert:
1. from the TCS signer cert
2. from the secret generated by attestation controller
Not sure which one should be the right.

Reply:
Does the secret generated by attestation controller means the certificate from KMRA? Which certificate do we use when doing manual verification?  I think `CACERT` should be TCS signer cert since TCS plays the role of a CA.

Reply:
confirmed, use the item 2